tests/test_dynamodb.py

```diff
@@ -4,10 +4,7 @@
 import datetime
 from dateutil import tz as tzutil
 from unittest.mock import MagicMock
-from c7n.testing import mock_datetime_now
```

Comment:
fwiw, just fyi, but we do use freeze gun library for this purpose now in other spots.